source/base/tensor_product_polynomials.cc

```diff
@@ -405,13 +406,26 @@ TensorProductPolynomials<dim, PolynomialType>::evaluate(
 
 
 
+template <int dim, typename PolynomialType>
+std::unique_ptr<ScalarPolynomialsBase<dim>>
+TensorProductPolynomials<dim, PolynomialType>::clone() const
+{
+  Assert(false, ExcNotImplemented());
+
+  return 0; // std_cxx14::make_unique<TensorProductPolynomials<dim,
+            // PolynomialType>>(*this);
```

Comment:
Do you mean on leaving this as a TODO for later?

Reply:
I was running into an error with this. Thanks for reminding me about this. I need to fix it but I'm unsure how.

Reply:
Post the error and we'll help you :-)

Reply:
Oh, it looks like I don't have that error now that the base class is accessible. That's nice!